parlai/scripts/train_model.py

```diff
@@ -444,10 +445,10 @@ def validate(self):
             or self.valid_optim * new_valid > self.valid_optim * self.best_valid
         ):
             print(
-                '[ new best {}: {}{} ]'.format(
+                '[ new best {}: {:.4g}{} ]'.format(
                     opt['validation_metric'],
                     new_valid,
-                    ' (previous best was {})'.format(self.best_valid)
+                    ' (previous best was {:.4g})'.format(self.best_valid)
```

Comment:
Nit: hmm is this two nested `.format()`s? Might be worth cleaning up with f-strings

Reply:
it's because the first go is different than others.